### PR TITLE
Fix script stuck flushing memcache.

### PR DESCRIPTION
**Pull request by MacWeber**

### CODE REVIEW

File: scripts/setup-local.sh [hunk 1]
```diff
@@ -6,8 +6,8 @@
 # Clear your local Memcached so old cache entries from the previous version of
 # the site don't blow up this release script. Clear this on both your local and
 # Vagrant, so we cover both use cases.
-echo "flush_all" | nc localhost 11211
-echo "flush_all" | nc localhost 11221
```
Comment (michaeldewolf85) [inline scripts/setup-local.sh]: Which project is this from?

File: scripts/setup-local.sh [hunk 2]
```diff
@@ -6,8 +6,8 @@
...
+print "flush_all\nquit" | nc localhost 11211
+print "flush_all\nquit" | nc localhost 11221
```
Comment (michaeldewolf85) [inline scripts/setup-local.sh]: Let's leave as `echo` that is POSIX compliant.

Comment (michaeldewolf85) [inline scripts/setup-local.sh]: Is it possible you don't have `nc` on your host?

**Comment by MarieKirya** [comment]
Something about this isn't quite right.

**Comment by MacWeber** [comment]
Fixed with option `-e`

**Pull request closed without merging**